schema/switch.yml

```diff
@@ -242,3 +242,294 @@ properties:
+    properties:
+      mvr-enable:
+        description: Enable/Disable MVR globally on the switch.
+        type: boolean
+        default: false
+        enum:
+          - true
+          - false
+      mvr-proxy-query-intvl:
+        description:
+          This command configures the interval (in seconds) at which the receiver port sends out general queries. 
+          The maximum value is determined based on 12 hours as maximum interval, and minimum as 1 second as allowed value.
+        type: integer
+        default: 125
+        maximum: 43200
+        minimum: 1
+      mvr-proxy-switching:
+        description:
+          Enable the MVR proxy switching mode, where the source port acts as a host, and the receiver port 
+          acts as an MVR router with querier service enabled.
+        type: boolean
+        default: false
+        enum:
+          - true
+          - false
+      mvr-robustness-val:
+        description:
+          Configure the expected packet loss, and thereby the number of times to generate report 
+          and group-specific queries when changes are learned about downstream groups, 
+          and the number of times group-specific queries are sent to downstream receiver ports. 
+          Right configuration ensures that multicast group memberships are correctly maintained 
+          even if some control messages are lost due to network issues.
+        type: integer
+        default: 2
+        maximum: 255
+        minimum: 1
+      mvr-source-port-mode:
+        description:
+          Configure the switch to forward only multicast streams that a source port has dynamically 
+          joined or to forward all multicast groups.
+        type: string
+        default: forward
+        enum:
+          - dynamic
+          - forward
+  mvr-domain-config:
+    description: Configure the Multicast VLAN Registration (MVR) domains.
+    type: array
+    items:
+      type: object
+      properties:
+        mvr-domain-id:
+          description: Unique identifier for a Multicast Domain defined under the MVR.
+          type: integer
+          minimum: 1
+          maximum: 10
+          default: 1
+        mvr-domain-enable:
+          description: Enable/disable Multicast VLAN Registration (MVR) for a specific domain.
+          type: boolean
+          default: false
+          enum:
+            - true
+            - false
+        mvr-domain-vlan-id:
+          description:
+            Per domain Level Multicast VLAN ID. Specifies the VLAN through which MVR multicast data is received. 
+            This is the VLAN to which all source ports must be assigned.
+          type: integer
+          minimum: 1
+          maximum: 4094
+          default: 1
+        mvr-domain-upstream-sip:
+          description: Configures the source IP address assigned to all MVR control packets sent upstream on all domains or on a specified domain.
+          type: string
+          format: ipv4
+          default: 0.0.0.0
```

Comment:
I wouldn't enforce default value (if omitted), especially to a reserved IP address;

Reply:
> I wouldn't enforce default value (if omitted), especially to a reserved IP address;

Agreed, there are already too many defaults in the overall schema definitions, sometimes this creates unintended consequences when provisioning things.

Reply:
Agree